app/src/main/java/org/mozilla/fenix/compose/SelectableChip.kt

```diff
@@ -84,12 +99,15 @@ private fun SelectableChipPreview() {
                 .background(FirefoxTheme.colors.layer1),
             horizontalArrangement = Arrangement.SpaceEvenly,
         ) {
-            SelectableChip(text = "Chirp", isSelected = false) { }
-            SelectableChip(text = "Chirp", isSelected = true) { }
+            SelectableChip(text = "ChirpOne", isSelected = false) {}
+            SelectableChip(text = "ChirpTwo", isSelected = true) {}
         }
     }
 }
 
+/**
+ * An example of a [SelectableChip] with custom colors
+ */
```

Comment:
Let's avoid comments for @Preview instead let's use `name` in the [@Preview annotation](https://developer.android.com/jetpack/compose/tooling/studio#preview) to provide more context in the design view previews.

An example here would be `@Preview(name = "Custom Colors")`.